patterns/19.async-nature-of-setState.md

```diff
@@ -112,11 +112,11 @@ var TestComponent = React.createClass({
     });
     console.log('State after (AJAX call): ' + JSON.stringify(this.state));
   }
-});
+};
 
 // Render to DOM
 ReactDOM.render(
-  React.createElement(TestComponent),
+  TestComponent,
```

Comment:
Not that experienced with this, but doesn't the component need to follow XML suit? 

```jsx
ReactDOM.render(
  <TestComponent />,
  document.getElementById('app')
);
```